pages/_app.tsx

```diff
@@ -26,6 +48,11 @@ function MyApp({ Component, pageProps }: AppProps): JSX.Element {
     )
     if (isHotjarCookiesAccepted === 'true') {
       hotjar.initialize(2392323, 6)
+      const userId = ''
+      const numberOfVisits = incrementVisits()
```

Comment:
I think this will run and increment the visits number on every page change, rather than visit. Maybe try adding an empty [dependency array to useEffect](https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects)?

Reply:
Good spot Nick 🕵🏻 That worked a treat!